library/src/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -814,6 +886,9 @@ private static boolean sameMonth(Calendar cal, MonthDescriptor month) {
   }
 
   private boolean isDateSelectable(Date date) {
+    if (isCheckDayInSelectableCals && !containsDate(selectableCals, date)) {
```

Comment:
I don't think we should make this change or add the additional `init` method above.  This is something that can and should be handled outside of the library.  When you're about to load the calendar, do the min/max calculation yourself, and then implement a custom `dateConfiguredListener` to handle the selectability of the dates you actually care about.  @JakeWharton thoughts?

The `containsDate`, `clearHighlightedDates`, and `scrollToDate` additional all seem fine to me.